podcasts/Onboarding/LoginCoordinator.swift

```diff
@@ -93,10 +104,45 @@ extension LoginCoordinator {
         authorizationController.performRequests()
     }
 
-    func signInWithGoogleTapped() { }
+    @MainActor
+    func signInWithGoogleTapped() {
+        guard let navigationController else {
+            return
+        }
+
+        Task {
+            progressAlert = SyncLoadingAlert()
+            do {
+                // First get token
+                try await self.googleSocialLogin.getToken(from: navigationController)
+
+                // If token is returned, perform login on our servers
+                progressAlert?.showAlert(navigationController, hasProgress: false, completion: nil)
```

Comment:
@emilylaguna I wasn't able to reproduce this. Can you please share the steps? Which request fails? The sync one? Or the login?

Reply:
@emilylaguna fixed in 36242b19bd5d2a6912b77f5e86ff2cbb534568b9

While I think this won't affect users it's good to have this in place just in case. What was happening was a race condition when presenting/dismissing the alert.